src/elements/content-sharing/SharingNotification.js

```diff
@@ -198,14 +199,15 @@ function SharingNotification({
     ]);
 
     // Set the collaborators list
-    const collaboratorsList: collaboratorsListType | null = useCollaborators(
-        api,
-        itemID,
-        itemType,
-        data => convertCollabsResponse(data, ownerEmail, currentUserID === ownerID),
-        () => handleError(contentSharingMessages.collaboratorsLoadingError),
+    const collaboratorsListFromAPI: Collaborations | null = useCollaborators(api, itemID, itemType, undefined, () =>
```

Comment:
Thoughts on having an options argument for the custom hook instead? That way you can avoid having to pass the `undefined` for the `successCallback`

Reply:
Good call. Fixed